sites/public/src/components/listing/ListingViewSeedsHelpers.tsx

```diff
@@ -0,0 +1,622 @@
+import {
+  EventType,
+  FieldGroup,
+  Form,
+  StandardTable,
+  StandardTableData,
+  t,
+  TableHeaders,
+} from "@bloom-housing/ui-components"
+import {
+  cloudinaryPdfFromId,
+  getOccupancyDescription,
+  getTimeRangeString,
+  occupancyTable,
+} from "@bloom-housing/shared-helpers"
+import { downloadExternalPDF } from "../../lib/helpers"
+import { CardContent, CardList } from "../../patterns/CardList"
+
+import styles from "./ListingViewSeeds.module.scss"
+
+export const getFilteredMultiselectQuestions = (
+  multiselectQuestions: ListingMultiselectQuestion[],
+  section: MultiselectQuestionsApplicationSectionEnum
+) => {
+  return multiselectQuestions.filter(
+    (elem) =>
+      elem.multiselectQuestions.applicationSection === section &&
+      !elem.multiselectQuestions.hideFromListing
+  )
+}
+
+export const getMultiselectQuestionData = (multiselectQuestions: ListingMultiselectQuestion[]) => {
+  return multiselectQuestions.map((listingMultiselectQuestion, index) => {
+    return {
+      ordinal: index + 1,
+      links: listingMultiselectQuestion?.multiselectQuestions?.links,
+      title: listingMultiselectQuestion?.multiselectQuestions?.text,
+      description: listingMultiselectQuestion?.multiselectQuestions?.description,
+    }
+  })
+}
+
+export const getAvailabilitySubheading = (waitlistOpenSpots: number, unitsAvailable: number) => {
+  if (waitlistOpenSpots) {
+    return `${waitlistOpenSpots} ${t("listings.waitlist.openSlots")}`
+  }
+  if (unitsAvailable) {
+    return `${unitsAvailable} ${
+      unitsAvailable === 1 ? t("listings.vacantUnit") : t("listings.vacantUnits")
+    }`
+  }
+  return null
+}
+
+export const getAvailabilityContent = (reviewOrderType: ReviewOrderTypeEnum) => {
+  switch (reviewOrderType) {
+    case ReviewOrderTypeEnum.waitlist:
+      return t("listings.waitlist.submitForWaitlist")
+    case ReviewOrderTypeEnum.firstComeFirstServe:
+      return t("listings.eligibleApplicants.FCFS")
+    default:
+      return t("listings.availableUnitsDescription")
+  }
+}
+
+export const hasMethod = (
+  applicationMethods: ApplicationMethod[],
+  type: ApplicationMethodsTypeEnum
+) => {
+  return applicationMethods.some((method) => method.type == type)
+}
+
+export const getMethod = (
+  applicationMethods: ApplicationMethod[],
+  type: ApplicationMethodsTypeEnum
+) => {
+  return applicationMethods.find((method) => method.type == type)
+}
+
+export const getPaperApplications = (applicationMethods: ApplicationMethod[]) => {
+  return (
+    getMethod(applicationMethods, ApplicationMethodsTypeEnum.FileDownload)
+      ?.paperApplications.sort((a, b) => {
+        // Ensure English is always first
+        if (a.language == "en") return -1
+        if (b.language == "en") return 1
+
+        // Otherwise, do regular string sort
+        const aLang = t(`languages.${a.language}`)
+        const bLang = t(`languages.${b.language}`)
+        if (aLang < bLang) return -1
+        if (aLang > bLang) return 1
```

Comment:
nitpick: you should be able to do a `localeCompare` 
```
return  t(`languages.${a.language}`).localeCompare(t(`languages.${b.language}`))
```

Reply:
This was existing (the majority of this file was) and I didn't audit this, but happy to update!